packages/web-components/src/fluent-design-system.ts

```diff
@@ -1,4 +1,4 @@
-import { DesignSystem } from "@microsoft/fast-foundation";
+import { DesignSystem } from '@microsoft/fast-foundation';
```

Comment:
This looks like a bad rebase, @czearing can you please check?

Reply:
I think this might just be Prettier running on the whole repo after a merge and adjusting things in other packages.